config/initializers/test_track_generate_timestamp.rb

```diff
@@ -0,0 +1,9 @@
+BUILD_TIMESTAMP_FILE_PATH = 'testtrack/build_timestamp'.freeze
+
+if Rails.env.test? || Rails.env.development?
+  TestTrack::BuildTimestamp = Time.zone.now.iso8601
+elsif File.exist?(BUILD_TIMESTAMP_FILE_PATH) && File.readable?(BUILD_TIMESTAMP_FILE_PATH) && !File.zero?(BUILD_TIMESTAMP_FILE_PATH)
+  TestTrack::BuildTimestamp = File.read(BUILD_TIMESTAMP_FILE_PATH)
```

Comment:
I think some basic validation of the file content is worthwhile, like is it in iso format. @jmileham does the iOS client do any validation?

Reply:
I'm not sure, but agreed. Also, this camelization isn't conventional for ruby - it looks like a class. We should either have a property of `TestTrack`, a la `TestTrack.build_timestamp`, or it should be using constant case, i.e. `TestTrack::BUILD_TIMESTAMP`. If we don't intend for it to be mutable, we should do that. Also, the timestamp file should probably be newline terminated, and if so, this should probably be chomping the results during its validation/normalization phase.